### ISSUE TITLE
invertObj: invert the keys and values of an object

### ISSUE THREAD

**Issue by jrTilak** [jrTilak/lazykit#56]

**Comment by ebe25** [comment]
hey @jrTilak can i get this assigned? 
What I have in mind that this has to be done  under objects right in registry?
`const obj = { a: 1, b: 2, c: 3 };
const result = invertObj(obj);
console.log(result);
// Expected output: { 1: a, 2: b, 3: c }`

**Comment by jrTilak** [comment]
Yes you are right. It's under objects and the keys will be values and values will be keys.

Just consider the edge cases like when the values are array or objects.
